akka-actor-tests/src/test/scala/akka/io/dns/AsyncDnsResolverIntegrationSpec.scala

```diff
@@ -181,6 +183,23 @@ class AsyncDnsResolverIntegrationSpec extends AkkaSpec(
       )
     }
 
+    "not hang when resolving raw IP address" in {
```

Comment:
I think these tests should be in the other test as they don't require a DNS server to run

Reply:
ok